### DISCUSSION TITLE
Bug Report: I cant put a song

### DISCUSSION THREAD

**Discussion by ActivePowah** [KadeArchive/Kade-Engine#364] [Q&A]
When I try to put a song with everything about it, it crashes

Here some photos
![Captura de pantalla 2021-05-03 203032](https://user-images.githubusercontent.com/76783222/116945653-9c9ea000-ac4e-11eb-88ae-60188558a539.png)
![Captura de pantalla 2021-05-03 203002](https://user-images.githubusercontent.com/76783222/116945655-9dcfcd00-ac4e-11eb-89f5-ce40a468089d.png)
![Captura de pantalla 2021-05-03 203128](https://user-images.githubusercontent.com/76783222/116945658-9e686380-ac4e-11eb-95ca-3ac69d1c09ef.png)
![Captura de pantalla 2021-05-03 203100](https://user-images.githubusercontent.com/76783222/116945661-9e686380-ac4e-11eb-95a0-276e5a28b889.png)

**Comment by ActivePowah** [comment]
I using kade engine 1.4.2

**Comment by isakube** [comment]
When does it crash? When you try to play the song?

**Reply by ActivePowah** [reply]
Yes
